Model/Customer/Notifier/MailSender.php

```diff
@@ -16,23 +16,34 @@
 use Magento\Framework\Mail\Template\TransportBuilder;
 use Magento\Store\Model\StoreManagerInterface;
 use Opengento\Gdpr\Model\Notifier\AbstractMailSender;
+use Psr\Log\LoggerInterface;
 
 final class MailSender extends AbstractMailSender implements SenderInterface
 {
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
+
     /**
      * @var View
      */
-    private View $customerViewHelper;
+    private $customerViewHelper;
 
-    private StoreManagerInterface $storeManager;
+    /**
+     * @var StoreManagerInterface
+     */
+    private $storeManager;
```

Comment:
The latest changes uses typed properties of php7.4. Please could you keep them? The composer.json php restriction is set to min php:7.4